presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2408,9 +2408,13 @@ else if (target instanceof DeleteHandle) {
 
     private static Function<Page, Page> enforceLayoutProcessor(List<Symbol> expectedLayout, Map<Symbol, Integer> inputLayout)
     {
-        int[] channels = expectedLayout.stream()
```

Comment:
you could use https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#peek-java.util.function.Consumer-
to validate values withing stream. Should be cleaner